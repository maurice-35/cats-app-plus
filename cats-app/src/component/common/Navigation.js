import React from 'react'
import { Navbar, Container, Nav } from "react-bootstrap";


const Navigation = () => {

	return (
		<Navbar expand="lg">
      <nav className="navbar fixed-top navbar-light bg-light">
        <Container fluid>
          <Navbar.Brand href="/Home">🐈</Navbar.Brand>
          <Nav.Link href="/cats">Cats</Nav.Link>
          <Nav.Link href="/Home">Home</Nav.Link>
          <Navbar.Brand href="/Home">🐈</Navbar.Brand>
        </Container>
      </nav>
    </Navbar>
	)
}

export default Navigation