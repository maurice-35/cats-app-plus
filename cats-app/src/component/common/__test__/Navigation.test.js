import { render, screen } from '@testing-library/react'
import Navigation from '../Navigation'

const mockedSetNavigation = jest.fn()

describe("Navigation", () => {
	it("should display cats page", () => {
		render(
			<Navigation
				cats={[]}
				setCats={mockedSetNavigation}
			/>
		)
		const clickedElement = screen.getByText(/Cats/i)
		expect(clickedElement).toMatchSnapshot()
	})
})