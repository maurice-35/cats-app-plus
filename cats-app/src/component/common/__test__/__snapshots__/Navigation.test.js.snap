// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should display cats page 1`] = `
<a
  class="nav-link"
  href="/cats"
>
  Cats
</a>
`;
